content/docs/releases/v1.0.0.md

```diff
@@ -0,0 +1,113 @@
++++
+title = "Release v1.0.0"
+short = "v1.0.0"
+weight = 1
+date = "2021-06-19"
++++
+
+NSM v1.0.0 is released via a set of [example use cases](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.0.0).
+
+## Kubernetes Compatibility
+NSM v1.0.0 has been successfully [integration tested](https://github.com/networkservicemesh/integration-k8s-kind/pull/304) with Kubernetes versions:
+- v1.21.1
+- v1.20.7
+- v1.19.11
+- v1.18.15
+- v1.17.11
+
+## Changes since version v0.2.0
+
+v1.0.0 major themes include:
+
+- [Payloads](#payloads)
+- [Wireguard](#wireguard)
+- [SRIOV](#sriov)
+- [Markdown Driven Testing](#markdown-driven-testing)
+- [Topology Aware Scale from Zero](#topology-aware-scale-from-zero)
+- [Latency Reduction](#latency-reduction)
+- [Refactoring](#refactoring)
+
+### Payloads
+
+Network Service Mesh intrinsically can support vWires that carry different payload types.
+Currently supported are:
+- IP
+- Ethernet
```

Comment:
This list does not look like a list in the web view.

Reply:
Fixed.